### ISSUE TITLE
[RVR supply box] and mount at the same time

### ISSUE THREAD

**Issue by Cekura** [WarEmu/WarBugs#20938]
**Expected behavior and actual behavior:**
Expected: You can only use one "spell" at a time.
Actual: You can grab boxes in RVR and have them equipped AFTER you use your mount.

**Steps to reproduce the problem:**
See top.

**Testing Screenshots/Videos/Evidences (always needed):**
[csysysg.webm](https://github.com/WarEmu/WarBugs/assets/134284254/adc6da5d-9819-4638-ae1f-364b6e826143)

**Comment by lilimx** [comment]
Hey,

What do you mean with "you can only use one spell at a  time"?

**Comment by Cekura** [comment]
Hello,

I was talking to a lot of other players and they never seen that. We
consider this a "spell stacking" and by this we mean using more than one
action by default.
Or is this intentional working this way?

To answer your question precisely:
This is posted in "Expected" ->
You should either only be able to mount or grab the box, not doing both at
a time (or am I wrong?)
So I'd expect not beeing able to spell stack.

I hope this will help you.
Cheers



Lilim ***@***.***> schrieb am Di., 30. Mai 2023, 22:39:

**Comment by lilimx** [comment]
Ah, I thought it was an error message that pops up.

**Comment by Cekura** [comment]
No, as you can see in the video I use the mount and then click the chest and the moment I mount up the box is applied to the character.
